### DISCUSSION TITLE
Maintenance Status

### DISCUSSION THREAD

**Discussion by milan-tom** [iSoron/uhabits#2081] [General Discussion] [answered]
I would just like to enquire about whether this repository is still actively maintained. I see a lot of unreviewed pull requests and it has been a while since the last commit. I'm not trying to discourage the maintainer(s). This is just a genuine enquiry about whether it is worth contributing to this repository or not.

**Comment by KyleSCraig** [comment]
I mean you're not likely to get a response from the people with merge rights until they come back. Hiqua was around in the middle of the year but looks like isoron hasn't been most of the year.

I've been hoping we might be able to do changes in a fork in a way that if iSoron decides to put more time into this, that they could be merged in, or if it ends up deviating enough over long enough time that it could get its own release at least on F-Droid.

**Comment by brog-io** [comment]
Seems like we are getting commits now

**Comment by brog-io** [comment]
@iSoron @hiqua

**Reply by hiqua** [reply]
Yeah sorry folks, maybe I can put some more time into it soon, it's tricky.

**Reply by brog-io** [reply]
> Yeah sorry folks, maybe I can put some more time into it soon, it's tricky.

Don't be sorry! You guys aren't getting paid for maintaining Loop, just a bit of communication would be nice.

**Comment by iSoron** [comment, marked as answer]
Yes, the project is maintained, but we cannot provide ETAs on anything (new releases, merging of PRs, etc) since all developers are volunteers working on their own free time, and have their own full-time jobs and families.
